App/screens/PinCreate.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import { Alert, Keyboard } from 'react-native'
+import * as Keychain from 'react-native-keychain'
+
+import { Button, TextInput, SafeAreaScrollView } from 'components'
+
+interface IPinCreate {
+  setupScreens: number
+  navigation: any
+}
+
+let textInput: any
+let secondTextInput: any
+
+function PinCreate(props: IPinCreate) {
+  const [pin, setPin] = useState('')
+  const [pinTwo, setPinTwo] = useState('')
+
+  const passcodeCreate = async (x: any) => {
+    const passcode = JSON.stringify(x)
+    const description = 'user authentication pin'
+    await Keychain.setGenericPassword(description, passcode, {
+      service: 'passcode',
+    })
+  }
+
+  const confirmEntry = (x: any, y: any) => {
+    if (x.length < 6 || y.length < 6) {
+      Alert.alert('Pin must be 6 digits in length')
+    } else if (x !== y) {
+      textInput.clear()
+      secondTextInput.clear()
+      Alert.alert('Pins entered do not match')
+    } else {
+      passcodeCreate(x)
+      props.navigation.navigate('Enter Pin')
```

Comment:
I think the user probably doesn't need to reenter their pin after they created it? So we should probably authenticate instead?